modules/ROOT/pages/alerts-on-runtime-manager.adoc

```diff
@@ -265,51 +259,55 @@ The following properties are available and can be used in the alert Email to, Su
 
 == Editing Existing Alerts
 
-If you view the panel for an application or server, you will see a *View Alerts* link. In there you will be able to see and manage the alerts that are sourced from it. You can also click on the *Alerts* link on the left panel to view all alerts together.
+If you view the panel for an application or server, you see a *View Alerts* link, where you can see and manage the alerts that are sourced from it. You can also click the *Alerts* link in the left panel to view a list of all alerts.
 
 image::view-alerts.png[alerts]
 
 
-From this menu you can filter the displayed alerts, click on an alert name to directly edit it, or click on the icon next to one to open its panel to view its settings. From this panel you can click the *edit* icon next to the name to edit the same options you have when creating a new one.
+From this menu you can filter the displayed alerts, click on an alert name to directly edit it, or click the icon next to one to open its panel to view its settings. From this panel you can click the *edit* icon next to the name to edit the same options you have when creating a new alert.
```

Comment:
"alert" instead of "one" for clarity?

Reply:
"to open its panel and view its settings"

Reply:
Point 1--I already removed "one" and just put alert.